src/desktop/apps/authentication/helpers.ts

```diff
@@ -126,7 +126,7 @@ export async function apiAuthWithRedirectUrl(
   response: Response,
   redirectPath: URL
 ): Promise<URL> {
-  const redirectUrl = sd.APP_URL + redirectPath.pathname
+  const redirectUrl = sd.APP_URL + redirectPath.pathname + redirectPath.search
```

Comment:
Fixes bug in our auth flow where query params were not being sent during a redirect